README.md

```diff
@@ -0,0 +1,120 @@
+| `dovecot_ldap_require_cert`       | `never`                 | When to require an LDAP certificate                                                      |
+| `dovecot_ldap_dn`                 | :heavy_check_mark:      | LDAP DN to bind with                                                                     |
+| `dovecot_ldap_dn_pass`            | :heavy_check_mark:      | Password of the LDAP DN to bind with                                                     |
+| `dovecot_ldap_base`               | :heavy_check_mark:      | LDAP search base                                                                         |
+| `dovecot_ldap_scope`              | `subtree`               | LDAP search scope                                                                        |
+| `dovecot_ldap_pass_attrs`         | `[uid=user]`            | LDAP-Dovecot attribute-mapping to maintain                                               |
+| `dovecot_ldap_pass_filter`        | :heavy_check_mark:      | LDAP filter for all users                                                                |
+| `dovecot_ldap_master_pass_filter` | `null`                  | LDAP filter for master users. Setting this to null disables the master feature           |
+| `dovecot_dsync`                   | `false`                 | Whether to enable the dsync functionality                                                |
+| `dovecot_dsync_max_connections`   | `10`                    | Maximum parallel dsync connections to spawn                                              |
+| `dovecot_dsync_port`              | `12345`                 | Dsync port to use                                                                        |
+| `dovecot_dsync_password`          | (:heavy_check_mark:)    | (Only if using dsync) Shared secret to establish dsync connection                        |
+| `dovecot_dsync_parameters`        | `-d -N -l 3000 -U`      | Parameters to set for dsync child processes                                              |
+| `dovecot_dsync_sync_timeout`      | `null`                  | Set to a non-null value to enable synchronous replication                                |
+| `dovecot_dsync_target`            | `null`                  | Dsync target to replicate with                                                           |
+| `dovecot_imap_hibernate_timeout`  | `5s`                    | Time for clients to idle before being pushed to the hibernation process                  |
+| `dovecot_imap_min_procs`          | `4`                     | Minimum amount of IMAP processes to keep available                                       |
+| `dovecot_private_prefix`          | `null`                  | Prefix for the private dovecot namespace                                                 |
+| `dovecot_private_mailboxes`       | `{}`                    | Mailbox configurations for the private mailboxes (see below)                             |
```

Comment:
Out of curiosity: Are there shared mailboxes? Because of the word private

Reply:
Maybe in the future. Not now as there is currently choice between shared mailboxes and dsync. You can't have both